src/main/java/hudson/plugins/brakeman/ResultSummary.java

```diff
@@ -29,7 +30,15 @@ public static String createSummary(final BrakemanResult result) {
         else {
             summary.append(Messages.Brakeman_ResultAction_MultipleWarnings(bugs));
         }
-        if (bugs > 0) {
+        if (ignoredBugs == 1) {
+            summary.append(". ");
+            summary.append(Messages.Brakeman_ResultAction_OneIgnoredWarning());

```

Comment:
StringBuilder has a fluent interface so calls to append return the StringBuilder instance so instead of:
```
summary.append(". ");
summary.append(Messages.Brakeman_ResultAction_OneIgnoredWarning());
```

you can do something like:
```
summary.append(". ").append(Messages.Brakeman_ResultAction_OneIgnoredWarning());
```
or if the line is too long, you can do:
```
summary.
    append(". ").
    append(". ");
```

the more of these method calls you chain together the bigger a difference it makes.  so it isn't going to be a huge improvement in this case because you are only chaining two of these calls together.  however, it is the preferred method of using these types of interfaces and does make it a little more readable.